### ISSUE TITLE
Help: Tabs/status bar blocked by site content

### ISSUE THREAD

**Issue by khawkins98** [matthewmx86/RetroThemesFirefox#4]
I suspect this isn't a bug, but a firefox config issue but I can't seem to guess what it might be. Have done a fair amoutn of research with no luck:

![image](https://user-images.githubusercontent.com/928100/227913031-2f609996-4c7e-466d-ac5d-2fcb1e72c61a.png)

You can see that the status bar/tabs are there (bottom right) but that the browser content is overlapping. 

Any ideas?

On Firefox 111.0 (new user of your theme, so not sure if this is a new issue). Manjaro Xfce (fully up to date).

As an aside, the  theme looked much better once I set `browser.tabs.inTitlebar` to `0`.

And thanks for this nice theme!
